### DISCUSSION TITLE
function name question

### DISCUSSION THREAD

**Discussion by xiaoqiuaming** [oceanbase/oceanbase#1373] [Q&A]
I think this function name is incorrect，it modified to ‘create_kv_if_need’ will be better
![image](https://user-images.githubusercontent.com/39131417/236609815-625a2c51-a875-4fd7-a281-50bb2b596b23.png)

**Comment by hnwyllmm** [comment]
It sounds interesting, but do you more information to support your opinion?

**Reply by xiaoqiuaming** [reply]
> It sounds interesting, but do you more information to support your opinion?

In this function, it will first get this value through key, if get failed then create new kv, if get succ directly return.
